x-pack/plugins/observability/docs/openapi/slo/components/schemas/slo_definition_response.yaml

```diff
@@ -0,0 +1,85 @@
+title: SLO definition response
```

Comment:
@kdelemme Out of curiosity under which url can I access the openapi docs?

Reply:
Ok I [see](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/docs/openapi/slo/entrypoint.yaml). I can simply make the [request](http://localhost:5601/s/default/api/observability/slos) and get the response. I thought we would have an html page with all the http requests that user can click on and see more information about the request and response.